docker-compose.security.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.8"
+
+services:
+  application:
+    image: public.ecr.aws/l6z6v3j6/digital-land-performance:${DOCKER_APPLICATION_TAG}
+    command: |
+      bash -c " \
+        sqlite3 digital-land.sqlite3 '.read digital-land-db-cleanup.sql' && \
```

Comment:
Can you elaborate on why we're running the security scan on a truncated database? Does the zapp scan runtime scale with database size?

Reply:
Yes it does, the ZAP spider job will spider every page it can possibly visit. Cutting the DB down reduces the duplication of requests. With an untruncated DB, we get ~200 pages, without truncating the DB, we hit around 8000 pages, which dramatically increases the length of time the active scan phase takes, from about 2 minutes to around 3 hours.